pulp_ansible/app/migrations/0028_manifest_and_files_json_fields.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.17 on 2020-12-10 15:04
+import json
+import tarfile
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+def _get_file_obj(tar, file_path, artifact_path):
+    for path in [file_path, f"./{file_path}"]:
+        try:
+            file_obj = tar.extractfile(path)
+        except KeyError:
+            file_obj = None
+        else:
+            break
+    if not file_obj:
+        raise FileNotFoundError(f"{file_path} not found")
+
+    return file_obj
+
+def set_manifest_and_files_json(apps, schema_editor):
+    CollectionVersion = apps.get_model("ansible", "CollectionVersion")
+    for collection_version in CollectionVersion.objects.all():
+        artifact = collection_version.contentartifact_set.get().artifact
+        with artifact.file.open() as artifact_file, tarfile.open(fileobj=artifact_file, mode="r") as tar:
+            collection_version.manifest = json.load(
+                _get_file_obj(tar, "MANIFEST.json", artifact.file.name)
+            )
+            collection_version.files = json.load(_get_file_obj(tar, "FILES.json", artifact.file.name))
+            collection_version.save()
```

Comment:
we can merge this with the data migration from https://github.com/pulp/pulp_ansible/pull/466